extension/src/plots/model/collect.test.ts

```diff
@@ -3,25 +3,31 @@ import omit from 'lodash.omit'
 import isEmpty from 'lodash.isempty'
```

Comment:
> The test coverage on the diff in this pull request is 91.2% (85% is the threshold).

I looked over the code and everything looked pretty covered. Any thing I could have missed to improve the coverage?